minecraft_servers/craftergang/manifest.json

```diff
@@ -88,5 +105,6 @@
     "country": "Germany",
     "country_code": "DE"
   },
+  "user_stats": "https://laby.net/@Kallecrafter",

```

Comment:
Thats your personal Laby.net page. `user_stats` url should be a page where each of your players can view their own statistics on your server.